src/DogStatsd.php

```diff
@@ -69,8 +73,9 @@ class DogStatsd
      * api_key and app_key
      *
      * @param array $config
+     * @param int   $decimal_precision
      */
-    public function __construct(array $config = array())
+    public function __construct(array $config = array(), $decimal_precision = 5)
```

Comment:
Yeah i'll change it to 2; if we dont support having this be configurable this takes away functionality from the client library since it seems DD support higher precision than 2 decimal places. I know internally, we would need to control precision.

Reply:
OK, then let's make that option part of the `$config` array to follow the same behavior that other option